api/v2/Routes/students/progressquerystring/index.js

```diff
@@ -1,11 +1,11 @@
 import { Router } from 'express';
+import {getEmailFromAuth} from "../../../../lib/googleAuthHelper.mjs";
 import {getMaxPointsSoFar, getStudentScores} from "../../../../lib/redisHelper.mjs";
 import ProgressReportData from '../../../../assets/progressReport/CS10.json' assert {type: 'json'};
-import 'express-async-errors';
```

Comment:
We should leave this in all of our routers until we can test it to make sure we don't need it.  It's for error handling.